NuGet.Docs/_Layout.cshtml

```diff
@@ -1,104 +1,111 @@
-        </div>
-        <div style="clear:both"></div>
-    </div>
-    <div class="content">
-        <section>
+        <div id="page-content-wrapper">
             @RenderBody()
-            <div style="clear:both;"></div>
-        </section>
+        </div>
     </div>
-    <footer class="page">
-        <p>
-            <a href="http://www.dotnetfoundation.org"><img src="~/images/dnf.png" alt=".NET Foundation" style="border-width:0" /></a>
-        </p>
-        <p>
-            &copy; 2011-@DateTime.UtcNow.Year .NET Foundation - <a href="//www.nuget.org/policies/terms">Terms of Use</a> - <a href="//www.nuget.org/policies/privacy">Privacy Policy</a> - <a href="http://status.nuget.org">Service status</a>
-        </p>
-        @if (Page.Source != null)
-        {
-            <p>
-                View or edit this page's source on GitHub:
-                <a href="https://github.com/NuGet/NuGetDocs/tree/master/NuGet.Docs/@HttpUtility.UrlPathEncode(Page.Source)" target="edit">https://github.com/NuGet/NuGetDocs/tree/master/NuGet.Docs/@HttpUtility.UrlPathEncode(Page.Source)</a>
-            </p>
-        }
-        else
-        {
-            <p>
-                View or edit this site's source on GitHub:
-                <a href="https://github.com/NuGet/NuGetDocs" target="edit">https://github.com/NuGet/NuGetDocs</a>
-            </p>
-        }
-        @RenderSection("footer", required: false)
-    </footer>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+    <script src="https://ajax.aspnetcdn.com/ajax/modernizr/modernizr-2.7.2.js"></script>
+    <script src="~/js/nuget.docs.js"></script>
+    <script src="~/Scripts/nanoscoller.min.js"></script>
+    <script src="~/scripts/highlight.pack.js"></script>
+    <script>
+        (function (i, s, o, g, r, a, m) {
+            i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+                (i[r].q = i[r].q || []).push(arguments)
+            }, i[r].l = 1 * new Date(); a = s.createElement(o),
+            m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+
+        ga('create', 'UA-30016961-3', 'nuget.org');
+        ga('send', 'pageview');
+    </script>
     <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        $("#menu-toggle").click(function (e) {
```

Comment:
Since these are executed on every page, can we move them into a `.js` file and reference that one instead?